mpd/commandlist.go

```diff
@@ -203,7 +223,7 @@ func (cl *CommandList) Delete(start, end int) error {
 	if end < 0 {
 		cl.cmdQ.PushBack(&command{fmt.Sprintf("delete %d", start), nil, cmdNoReturn})
 	} else {
-		cl.cmdQ.PushBack(&command{fmt.Sprintf("delete %d %d", start, end), nil, cmdNoReturn})
+		cl.cmdQ.PushBack(&command{fmt.Sprintf("delete %d:%d", start, end), nil, cmdNoReturn})
```

Comment:
Non-actionable comment: It's unfortunate we don't have a good tests for command lists. Otherwise, I'd ask you to add a regression test for this fix.